job-server/src/main/scala/spark/jobserver/BinaryManager.scala

```diff
@@ -40,7 +40,7 @@ class BinaryManager(jobDao: ActorRef) extends InstrumentedActor {
   import scala.concurrent.duration._
   import akka.pattern.{ask, pipe}
   import context.dispatcher
-  implicit val daoAskTimeout = Timeout(3 seconds)
+  implicit val daoAskTimeout = Timeout(60 seconds)
```

Comment:
When we test mesos cluster mode, we often got Timeout error, because the jar we submit is a little big(about 30M) and our database is not fast enough,  and there are many synchronize call in spark-jobserver like `jobDao ? JobDAOActor.SaveBinary(appName, binaryType, uploadTime, binBytes)).` and `val resp = (jobDao ? JobDAOActor.GetApps(filterOpt)).mapTo[JobDAOActor.Apps]
      resp.map { msg => msg.apps } pipeTo requestor`, I simply enlarge the daoAskTimeout, but it is better to provide a config for this timeout.